### ISSUE TITLE
tensor shape for conv2 weights on tflite files conflicts with non-quantized graph

### ISSUE THREAD

**Issue by jnorwood** [lutzroeder/netron#154]
In the mobilenet_v1 non-quantized 224 at this link, the filter properties displays the data format as NHWC, and displays  the shape for the first convolution filter as float32[3,3,3,32] in the frozen.pb file

https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet_v1.md

At the same link is a quantized version of mobilenet v1 224.  Its tflite file does not display a data format, but the weights tensor of the first convolution displays as byte[32,3,3,3].  

However this github page say tflite  supports only NHWC... so I would expect both of these models to be using  the same shape for storage.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/tf_ops_compatibility.md

So, is this some error in the way the models are being stored, or is it the viewer?
Thanks...

**Comment by lutzroeder** [comment]
Can you investigate this further?

```
node {
  name: "MobilenetV1/Conv2d_0/weights"
  op: "Const"
  attr {
    key: "value"
    value {
      tensor {
        dtype: DT_FLOAT
        tensor_shape {
          dim {
            size: 3
          }
          dim {
            size: 3
          }
          dim {
            size: 3
          }
          dim {
            size: 32
          }
        }
```

```
0 | MobilenetV1/Conv2d_0/weights | FLOAT32 | [32, 3, 3, 3] | 32 | {}
```

**Comment by jnorwood** [comment]
ok, I investigated further.  It appears to me to be that tflite weight data is being persisted in .tflite with an order that differs from the persistence in the .pb, and the tflite shape is also does not match the weight shape specified in their conv2d documentation.  I added an issue at tensorflow.  I think your viewer is displaying the tflite data as it is specified in the weight's shape parameter.
https://github.com/tensorflow/tensorflow/issues/22540
